roles/requirements.yml

```diff
@@ -2,4 +2,4 @@
 - name: cloudalchemy.prometheus
 - name: cloudalchemy.grafana
 - name: geerlingguy.certbot
-- name: nginxinc.nginx
\ No newline at end of file
+- name: community.general
```

Comment:
Sollten wir auch in die Readme aufnehmen dass das mit der Galaxy angezogen wird und vielleicht ein Satz dazu was das tut.

Reply:
I added it to the readme and put `(node)` as description for what we need it.